department-of-veterans-affairs/va.gov-cms
# VACMS-2696: Improve facility health service listing on facility pages.

## Description

See #2696 

Remove unneeded columns from facility services, and add destination parameter to edit link"


## Testing done

Manual

## Screenshots
Before

![image](https://user-images.githubusercontent.com/643678/90557709-f1ee5c00-e168-11ea-97d0-42c3b7704431.png)


After

![VA_Pittsburgh_Health_Care___Pittsburgh_VA_Medical_Center-University_Dr](https://user-images.githubusercontent.com/643678/90557765-03cfff00-e169-11ea-83f4-1060c9ff834e.jpg)


## QA steps

1. Visit /pittsburgh-health-care/locations/pittsburgh-va-medical-center-university-drive
2. Check that there are no fields after the Last updated field.
3. Click edit on any service. 
4. There should be a destination parameter that takes you back to the node you were just looking at.
